modin/data_management/factories.py

```diff
@@ -18,6 +18,10 @@
 from modin import __execution_engine__ as execution_engine
 from modin import __partition_format__ as partition_format
 
+import pandas
```

Comment:
I don't think these are getting used at the moment. We should remove them if they are not used.

Reply:
Pandas name is used on the next line to define `pandas.CategoricalDtype`.